include_core/omrformatconsts.h

```diff
@@ -0,0 +1,182 @@
+#ifndef OMRFORMATCONSTANTS_H_
```

Comment:
This file needs a copyright header. :slightly_smiling_face:

Reply:
Added, thanks.